pkg/channel/distributed/dispatcher/dispatcher/dispatcher.go

```diff
@@ -283,7 +283,7 @@ func (d *DispatcherImpl) ConfigChanged(configMap *v1.ConfigMap) Dispatcher {
 	// Merge The ConfigMap Settings Into The Provided Config
 	saramaSettingsYamlString := configMap.Data[testing.SaramaSettingsConfigKey]
 
-	newConfig, err := client.MergeSaramaSettings(nil, saramaSettingsYamlString)
+	newConfig, err := client.BuildSaramaConfig(nil, saramaSettingsYamlString, nil)
```

Comment:
If we have the Kafka auth in channel CRD for the distributed channel in the future, we can just pass it here.